CHANGELOG.md

```diff
@@ -58,6 +58,10 @@
   Salt dependencies from "base" RHEL 7 repository
   (PR [#3083](https://github.com/scality/metalk8s/pull/3083))
 
+- [#3128](https://github.com/scality/metalk8s/issues/3128) - Do not consider all ISOs
+  mounted in `/srv/scality` as MetalK8s mountpoint
```

Comment:
```suggestion
- [#3128](https://github.com/scality/metalk8s/issues/3128) - No longer assume ISOs
  mounted under `/srv/scality` are Solutions (now validating the presence and format
  of a `manifest.yaml` file)
```

(not sure if we want to specify the part about the manifest though, feel free to omit it)

Reply:
Indeed to me we do not really care about "manifest.yaml" here but I will update the sentence